pkg/server/image_list.go

```diff
@@ -23,22 +23,24 @@ import (
 )
 
 // ListImages lists existing images.
-// TODO add filters
-// TODO harden api
+// TODO(mikebrow): add filters
+// TODO(mikebrow): harden api
 func (c *criContainerdService) ListImages(ctx context.Context, r *runtime.ListImagesRequest) (*runtime.ListImagesResponse, error) {
 	imageMetadataA, err := c.imageMetadataStore.List()
 	if err != nil {
 		return nil, err
```

Comment:
nit: fmt.Errorf("failed to list image metadata from store: %v", err)

And also add detailed error reason for some following errors.